web/app.js

```diff
@@ -2249,9 +2292,17 @@ function webViewerKeyDown(evt) {
           handled = true;
         }
         break;
-      case 13: // enter key
       case 40: // down arrow
       case 34: // pg down
+        if (!pdfViewer.isVerticalScrollbarEnabled) {
```

Comment:
My comment above also applies here (the explanation is likely similar).

Reply:
Same!

Reply:
Same too!